freqtrade/strategy/interface.py

```diff
@@ -742,6 +762,16 @@ def advise_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
         :return: a Dataframe with all mandatory indicators for the strategies
         """
         logger.debug(f"Populating indicators for pair {metadata.get('pair')}.")
+
+        # call populate_indicators_Nm() which were tagged with @informative decorator.
+        for attr_name in dir(self.__class__):
+            cls_attr = getattr(self.__class__, attr_name)
+            if not callable(cls_attr):
+                continue
+            if not getattr(cls_attr, '_is_informative', False):
+                continue
+            dataframe = cls_attr(self, dataframe, metadata)
+
```

Comment:
that's gonna have the same problem than the `populate_indicators` we looked at.

If you need to do "additional" stuff in methods that users implement by default (this has been in all templates for years), then you'll need to create a wrapper for this where the additional stuff is done - and then (or before?) calls the exisitng method.


consider the most likely scenario for this enhancement to avoid having it break stuff
* i have a working strategy (take sample_strategy) - with 1 additional pair added to informative_pairs
* I see this change - and wanna use it for the 2nd pair i need
* I just add a decorated method as shown / documented - without touching my already existing implementation for now

Reply:
Good point 👍🏻 i ammended commit, creating internal `gather_informative_pairs()` which does extra logic needed. Now we can mix and match decorators with old way of doing things 🚀

Reply:
i knew you'd understand me even if i comment it on the wrong lines :laughing: